examples/programmer-art/src/index.mjs

```diff
@@ -1,10 +1,52 @@
 import * as c from '@reach-sh/simulator-client';
 import cytoscape from 'cytoscape';
 import klay from 'cytoscape-klay';
+import Prism from 'prismjs';
```

Comment:
You gotta use Shiki, that what we use in the docs --- https://github.com/reach-sh/reach-lang/tree/master/docs/dev

Reply:
i would rather not use this. this library seems to be in a terribly broken state. just trying to run the basic demo i get half a dozen errors including these issues that have been open for months : 

- https://github.com/shikijs/shiki/issues/216
- https://github.com/shikijs/shiki/issues/246

Reply:
We really can't maintain multiple different formatting engines for Reach though. I'll let this slide, but make a note to revisit.